packages/console/src/pages/GetStarted/index.tsx

```diff
@@ -1,110 +1,147 @@
+import classNames from 'classnames';
+import { useCallback, useContext, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
-import CompleteIndicator from '@/assets/icons/task-complete.svg';
+import CheckPreviewDark from '@/assets/icons/check-demo-dark.svg';
+import CheckPreview from '@/assets/icons/check-demo.svg';
+import CreateRoleDark from '@/assets/icons/create-role-dark.svg';
+import CreateRole from '@/assets/icons/create-role.svg';
+import SocialDark from '@/assets/icons/social-dark.svg';
+import Social from '@/assets/icons/social.svg';
 import PageMeta from '@/components/PageMeta';
-import Button from '@/ds-components/Button';
+import { ConnectorsTabs } from '@/consts';
+import { AppDataContext } from '@/contexts/AppDataProvider';
+import { LinkButton } from '@/ds-components/Button';
 import Card from '@/ds-components/Card';
-import ConfirmModal from '@/ds-components/ConfirmModal';
-import DynamicT from '@/ds-components/DynamicT';
 import Spacer from '@/ds-components/Spacer';
+import TextLink from '@/ds-components/TextLink';
 import useTenantPathname from '@/hooks/use-tenant-pathname';
-import useUserPreferences from '@/hooks/use-user-preferences';
+import useTheme from '@/hooks/use-theme';
+
+import CreateForm from '../Applications/components/CreateForm';
+import GuideCard, { type SelectedGuide } from '../Applications/components/GuideCard';
+import useAppGuideMetadata from '../Applications/components/GuideLibrary/hook';
 
 import FreePlanNotification from './FreePlanNotification';
-import Skeleton from './components/Skeleton';
-import useGetStartedMetadata from './hook';
 import * as styles from './index.module.scss';
 
 function GetStarted() {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { navigate } = useTenantPathname();
-  const { data, isLoading } = useGetStartedMetadata();
-  const { update } = useUserPreferences();
-  const [showConfirmModal, setShowConfirmModal] = useState(false);
-  const [isUpdating, setIsUpdating] = useState(false);
-
-  const hideGetStarted = async () => {
-    if (isUpdating) {
-      return;
-    }
-    setIsUpdating(true);
+  const { tenantEndpoint } = useContext(AppDataContext);
+  const [selectedGuide, setSelectedGuide] = useState<SelectedGuide>();
+  const [_, getStructuredMetadata] = useAppGuideMetadata();
+  const [showCreateForm, setShowCreateForm] = useState<boolean>(false);
+  const theme = useTheme();
+  const isLightMode = theme === Theme.Light;
+  const PreviewIcon = isLightMode ? CheckPreview : CheckPreviewDark;
+  const SocialIcon = isLightMode ? Social : SocialDark;
+  const RbacIcon = isLightMode ? CreateRole : CreateRoleDark;
 
-    try {
-      await update({ getStartedHidden: true });
-      // Navigate to next menu item
-      navigate('/dashboard');
-    } finally {
-      setIsUpdating(false);
-    }
-  };
+  const featuredAppGuides = useMemo(
+    () => getStructuredMetadata().featured.slice(0, 4),
```

Comment:
better to leave a comment here for why we slice for 4 items

Reply:
comment added